libs/api/domains/education/src/lib/education.service.ts

```diff
@@ -24,7 +45,105 @@ export class EducationService {
   async downloadPdfLicense(
     nationalId: string,
     licenseId: string,
-  ): Promise<Response> {
-    return this.mmsApi.downloadLicensePDF(nationalId, licenseId)
+  ): Promise<string | null> {
+    const responseStream = await this.mmsApi.downloadLicensePDF(
+      nationalId,
+      licenseId,
+    )
+
+    return this.s3Service.uploadFileFromStream(responseStream, {
+      fileName: uuid(),
+      bucket: this.config.fileDownloadBucket,
+    })
+  }
+
+  async getFamily(nationalId: string) {
+    const family = await this.nationalRegistryApi.getMyFamily(nationalId)
+    return family.filter(
```

Comment:
The nationalregistry api domain has this logic already, but we need to move it to the nationalregistry client so it can be reused here. I want to do that in another slice.